ui/framework/src/ui-framework/accudraw/AccuDrawCommandItems.ts

```diff
@@ -141,4 +141,12 @@ export class AccuDrawCommandItems {
       execute: () => AccuDrawShortcuts.defineACSByPoints(),
     });
   }
+

```

Comment:
Same comment... this could be an immediate Tool and the CommandItemDef changed to a ToolItemDef.

Reply:
These CommandItemDefs are referencing AccuDrawShortcuts in core. AccuDraw works with the current tool. I don't know how expensive it would be to run an immediate Tool to change an AccuDraw setting or mode.

Reply:
I don't believe immediate commands do anything more the run a function in the end. They do not interrupt the current tool, it just really gives you key-in access to a method and an easy way to pass arguments.